.drone/drone.jsonnet

```diff
@@ -92,6 +98,43 @@ local docker_manifest(app) = {
   },
 };
 
+local deploy_to_dev() = {
+  image: 'us.gcr.io/kubernetes-dev/drone/plugins/updater',
+  settings: {
+    config_json: |||
+        {
+          "destination_branch": "master",
+          "pull_request_branch_prefix": "cd-tempo-dev",
+          "pull_request_enabled": false,
+          "pull_request_team_reviewers": [
+            "tempo"
+          ],
+          "repo_name": "deployment_tools",
+          "update_jsonnet_attribute_configs": [
+            {
+              "file_path": "ksonnet/environments/tempo/dev-us-central-0.tempo-dev-01/images.libsonnet",
+              "jsonnet_key": %s,
+              "jsonnet_value_file": ".tags"
+            }
+            {
+              "file_path": "ksonnet/environments/tempo/dev-us-central-0.tempo-dev-01/images.libsonnet",
+              "jsonnet_key": %s,
+              "jsonnet_value_file": ".tags"
+            }
+            {
+              "file_path": "ksonnet/environments/tempo/dev-us-central-0.tempo-dev-01/images.libsonnet",
+              "jsonnet_key": %s,
+              "jsonnet_value_file": ".tags"
+            }
+          ]
```

Comment:
I _think_ this can be simplified with a for loop:

```jsonnet
          "update_jsonnet_attribute_configs": [
            {
              "file_path": "ksonnet/environments/tempo/dev-us-central-0.tempo-dev-01/images.libsonnet",
              "jsonnet_key": app,
              "jsonnet_value_file": ".tags"
            }
            for app in ['tempo', 'tempo-query', 'tempo-vulture'],
          ]
```

But we would need to make this an object as well then instead of a string (`|||`)

Reply:
Thanks 👍